docs/rst/reference/stop.rst

```diff
@@ -0,0 +1,13 @@
+stop
+=======
+
+**New in version 0.2.0**
+
+.. program:: ansible-container stop [service [service ...]]
+
+Stop running containers for the services defined in *container.yml.* Optionally list one or more services to stop. The name of the service must match a service defined in
```

Comment:
Markup container.yml in both places, and put the sentence-ending period outside of the markup.